shared/data/registry.json

```diff
@@ -1,4 +1,30 @@
 {
+  "chroma": {
```

Comment:
adding non-services into the services area of this file is going to cause issues later when we automate the list of services from dash.